app/models/communitarian/resolution.rb

```diff
@@ -21,9 +21,23 @@ def reopen_weekly_resolution!(original_post)
       post.save_custom_fields(true)
       poll = post.reload.polls.first
       poll.update!(close_at: resolution_schedule.next_close_time)
+      self.reorder_resolutions!(original_post.topic)
       self.schedule_jobs(post)
     end
 
+    def reorder_resolutions!(topic)
+      resolutions = topic.posts.select { |p| p.custom_fields["is_resolution"] }
```

Comment:
we need to update sort_order for all posts in the topic because we adding the new post to the beginning of the list.

We can do it in one sql request:
```
topic.posts.with_deleted.update_all("sort_order=sort_order+1")
```

Reply:
I think it works since we have comments in resolutions topics